edge-apps/clock/screenly.yml

```diff
@@ -1,15 +1,15 @@
+---
 app_id: ''
```

Comment:
Please remove this line as well everywhere

Reply:
Just a clarification - app_id can't be empty in the manifest. It should be not present at all or have a proper non-empty id.